crates/apollo-compiler/src/database/ast.rs

```diff
@@ -86,7 +86,7 @@ mod tests {
 
         assert_eq!(ast.recursion_limit().high, 2);
         assert_eq!(ast.errors().len(), 1);
-        assert_eq!(ast.document().definitions().into_iter().count(), 2);
+        assert_eq!(ast.document().definitions().into_iter().count(), 1);
```

Comment:
no clue why this supposedly had 2 definitions! clearly there is only one. oh well \o/